pkg/validation/app.go

```diff
@@ -97,6 +103,58 @@ func (v *Validator) validateConfig(ctx context.Context, cr v1alpha1.App) error {
 	return nil
 }
 
+func (v *Validator) validateMetadataConstraint(ctx context.Context, cr v1alpha1.App) error {
+	name := key.AppCatalogEntryName(key.CatalogName(cr), key.AppName(cr), key.Version(cr))
+
+	entry, err := v.g8sClient.ApplicationV1alpha1().AppCatalogEntries(metav1.NamespaceDefault).Get(ctx, name, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
+		v.logger.Debugf(ctx, "appCatalogEntry %#q not found, skipping validation", name)
```

Comment:
```suggestion
		v.logger.Debugf(ctx, "appcatalogentry %#q not found, skipping metadata validation", name)
```

I think the logs are easier to read in lower case.